docs/components/plugins/AppConfigPropsTable.js

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { css } from 'react-emotion';
+import { InlineCode } from '~/components/base/code';
+import MDX from '@mdx-js/runtime';
+import * as components from '~/common/translate-markdown';
+
+const STYLES_TABLE = css`
+  font-size: 1rem;
+  margin-top: 24px;
+`;
+
+const STYLES_DESCRIPTION_CELL = css`
+  word-break: break-word;
+  white-space: break-spaces;
+`;
+
+//appends a property and recursivley calls itself to append sub-properties, accounting for nested "level"
+function appendProp(formattedSchema, property, _level) {
```

Comment:
```suggestion
function appendProperty(formattedSchema, property, _level) {
```

I'm a big fan of slightly longer function names to make them more readable. Up to you if you agree.